implementations/rust/ockam/ockam_api/src/cloud/share/create.rs

```diff
@@ -0,0 +1,163 @@
+use minicbor::{Decode, Encode};
+use serde::Serialize;
+
+use super::*;
+
+#[derive(Clone, Debug, Decode, Encode, Serialize)]
+#[cbor(map)]
+#[rustfmt::skip]
+pub struct CreateInvite {
+    #[n(1)] pub expires_at: Option<String>,
+    #[n(2)] pub grant_role: RoleInShare,
+    #[n(3)] pub recipient_email: Option<String>,
+    #[n(4)] pub remaining_uses: Option<usize>,
+    #[n(5)] pub scope: ShareScope,
+    #[n(6)] pub target_id: String,
+}
+
+#[derive(Clone, Debug, Decode, Encode, Serialize)]
+#[cbor(map)]
+#[rustfmt::skip]
+pub struct CreateServiceInvite {
```

Comment:
In terms of vocabulary I understand that `invite` is a more informal than `invitation`. Would `invitation` be better?

Reply:
I've attempted to moved all wording, module names, commit messages, etc. to use invitation rather than invite. I'll only continue to use the latter when needed as a verb.